From: Zhiyi Zhang <zzhang@codeweavers.com>
Subject: [PATCH] gdi32: Quiet a noisy FIXME.
Message-Id: <36ee2379-e33d-54e5-209c-07c094f8fd96@codeweavers.com>
Date: Mon, 21 Jun 2021 17:08:46 +0800

"fixme:font:get_name_record_codepage encoding 20 not handled, platform 1." always shows up.

Signed-off-by: Zhiyi Zhang <zzhang@codeweavers.com>
---
 dlls/gdi32/opentype.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dlls/gdi32/opentype.c b/dlls/gdi32/opentype.c
index 5a22f61cbd3..069ffaabf56 100644
--- a/dlls/gdi32/opentype.c
+++ b/dlls/gdi32/opentype.c
@@ -344,7 +344,7 @@ static UINT get_name_record_codepage( enum OPENTYPE_PLATFORM_ID platform, USHORT
         case TT_NAME_MAC_ENCODING_THAI:
             return 10021;
         default:
-            FIXME( "encoding %u not handled, platform %d.\n", encoding, platform );
+            WARN( "encoding %u not handled, platform %d.\n", encoding, platform );
             break;
         }
         break;

-- 
2.30.2

